### DISCUSSION TITLE
Efficientnet b5-b7 model weights availability in candle

### DISCUSSION THREAD

**Discussion by relf** [huggingface/candle#2221] [Q&A] [answered]
Hi, I've ran successfully efficientnet example and noticed that [b5-b7 models are not available](https://huggingface.co/lmz/candle-efficientnet/tree/main). 

What is the best way to get the missing models in candle? Export from pytorch with onnx and load in candle? Thanks for your help.

**Comment by LaurentMazare** [comment, marked as answer]
Not really sure why I hadn't uploaded these models at that point but I've just added them to the repo you mentioned together with a very simple export script.

**Comment by relf** [comment]
Thank you for your answer. And I've just found [your comment here](https://github.com/huggingface/candle/issues/1075#issue-1939193283) very useful to grasp the porting process of a PyTorch model to candle.
